decidim/decidim
# Fix initiatives signatures issues

#### :tophat: What? Why?
As described in #6862, initiatives offline signatures were not being shown and it was not possible to accept initiatives even when the goal was reached. It turns that the second issue is not related to the offline signatures, but to another bug related to the way iniative types are used. This PR fixes both.

#### :pushpin: Related Issues
- Fixes #6862

#### Testing
1. Create and publish an in-person or mixed initiative.
2. Update the number of in-person signatures (and set a past date as the "End of signature period".
3. Check that the admin page shows the "Accept initiative" button.
4. Visit the initiative page and check that the number of in-person signatures is included in the total count of signatures received by the initiative.

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
